### DISCUSSION TITLE
How do I allow a group of people to approve PRs but let all with write access merge?

### DISCUSSION THREAD

**Discussion by cannontrodder** [community/community#21918] [New to GitHub]
<p>I have a repo owned by our organisation and I have used the CODEOWNERS file to specify that any change to a file under /policies/* must be approved by a member of our infosec team:</p>
<p>policies <span class="mention">@tombolaltd</span>/infosec-team</p>
<p>This then tells me that I need an approved reviewer from our infosec-team - once I get that, my colleague gets the below screen. It says he must be a collaborator and he must have write access. We added that person to the repo as a member of a team and that team has write access. Adding everyone as a collaborator is not what we want to do. This person should be able to merge, can someone help with where we may be going wrong?</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/fbc3791b4c9fbaab5d54b034a285efee3666542b.jpeg" data-download-href="/uploads/short-url/zVcDWldcyJgyLhMM2AwRVimkomT.jpg?dl=1" title="github.png" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/6507976/181067523-4d9aa544-4cfe-4bfb-9f5e-a0caab351a8e.jpg" alt="github.png" data-base62-sha1="zVcDWldcyJgyLhMM2AwRVimkomT" width="669" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/fbc3791b4c9fbaab5d54b034a285efee3666542b_2_669x500.jpg, https://aws1.discourse-cdn.com/github/optimized/1X/fbc3791b4c9fbaab5d54b034a285efee3666542b_2_1003x750.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/fbc3791b4c9fbaab5d54b034a285efee3666542b_2_1338x1000.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/fbc3791b4c9fbaab5d54b034a285efee3666542b_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github.png</span><span class="informations">1559×1165 123 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by lee-dohm** [comment, marked as answer]
<p>Since you’re asking about configuration on a private repository and, I suspect, non-public teams and organization members, you’ll need to contact GitHub Support directly so that they can take a look at the private organization details. You can contact GitHub Support through the contact form at <a href="https://github.com/contact" rel="nofollow noopener">https://github.com/contact</a></p>
<p>I hope that helps!</p>
